### ISSUE TITLE
Mod taking too long to load, leading to timeout crash on Aternos.

### ISSUE THREAD

**Issue by Luningor** [The-Fireplace/Mob-Rebirth#17]
**Description**
Trying to start a server  with custom mob config causes the server to crash.

**To Reproduce**
1. Create a custom mob config file.
2. Copy it to a server, under /config/mobrebirth_custommobsettings/minecraft/mob.json
3. Start server.

**Logs/Screenshots/Videos**
https://pastebin.com/8Q0YKGzG
Mirror: https://mclo.gs/FKud1L6

**Versions:**
 - Minecraft: [1.17.1 (0.11.7)]
 - Mod Version: [10.2.0]
 - Fabric API: [Fabric API 0.40.0]

**Comment by The-Fireplace** [comment]
I'm having difficulty reproducing the issue. Are those the full logs?

**Comment by Luningor** [comment]
> I'm having difficulty reproducing the issue. Are those the full logs?

Yes, they are, sadly. If it helps, i could provide a list of the other mods I use alongside Reborn. I discarded the idea since it only screwed up when I had said files on the folder; yet it could be a conflict between two of these mods.

*The mods*
antighost-1.17.1-fabric0.36.1-1.1.4.jar
AxesAreWeapons-1.3.1-forge-1.17.1.jar
cloth-config-5.0.38-fabric.jar
fabric-api-0.40.1+1.17.jar (Tried updating, didn't work)
ferritecore-3.1.0-fabric.jar
lithium-fabric-mc1.17.1-0.7.4.jar
more-potions-1.0.1.jar
quickshulker-1.2.3-1.17.jar
shulkerboxtooltip-3.0.1+1.17.jar
skin-restorer-1.0.4.jar
